docs/root/configuration/http/http_filters/original_src_filter.rst

```diff
@@ -73,3 +73,5 @@ The following example configures Envoy to use the original source for all connec
         "@type": type.googleapis.com/envoy.extensions.filters.listener.original_src.v3.OriginalSrc
         mark: 123
     - name: envoy.filters.http.router
+      typed_config:
```

Comment:
these really need to go in yaml files so they are properly tested

Reply:
Maybe we should add a warning for YAML code-samples greater than, say, 20 lines?